charactersheet/charactersheet/viewmodels/dm/encounter.js

```diff
@@ -14,7 +14,8 @@ function EncounterViewModel() {
 
     self.sections = [
         { property: 'pointOfInterestSectionViewModel', vm: PointOfInterestSectionViewModel, model: PointOfInterestSection },
-        { property: 'notesSectionViewModel', vm: NotesSectionViewModel, model: NotesSection }
+        { property: 'notesSectionViewModel', vm: NotesSectionViewModel, model: NotesSection },
+        { property: 'npcSectionViewModel', vm: NPCSectionViewModel, model: NPCSection }
```

Comment:
Can do. Can we define the order somewhere so that it can be respected going forward?